.github/workflows/manual-docker-publish.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: Manual - Publish Docker Images
+on:
+  workflow_dispatch:
+    inputs:
+      node:
+        default: "xeon"
+        description: "Hardware to run test"
+        required: true
+        type: string
+      tag:
+        default: "rc"
+        description: "Tag to publish, like [1.0rc]"
+        required: true
+        type: string
+      publish_tags:
+        default: "latest,1.x"
+        description: "Tag list apply to publish images, like [latest,1.0]"
+        required: false
+        type: string
+
+permissions: read-all
+jobs:
+  publish:
+    strategy:
+      matrix:
+        image: ["studio-frontend", "studio-backend", "app-frontend", "app-backend"]
```

Comment:
Support to set the image list as an input param, sometimes you may want to only refresh one of the images.

Reply:
updated support for having the image as boolean input parameter